### PR TITLE
Add Guidestar seal

### PR DESCRIPTION
**Pull request by ludstuen90**
This pull request will add the Guidestar seal to the Hacks/hackers homepage.

### CODE REVIEW

File: themes/hackshackers-2017/layouts/partials/home-signup.html [hunk 1]
```diff
@@ -4,4 +4,8 @@ <h4 class="heading-section">Sign Up</h4>
 		{{ partial "utils/mailchimp-form.html" }}
 		{{ . }}
 	</div>
+        <div class="right-column-signup">
+           <h4 class="heading-section">Credibility</h4>
```
Comment (burtherman) [inline themes/hackshackers-2017/layouts/partials/home-signup.html:8]: Not sure if we need this large header with "Credibility" - can we remove this and just have the text?

Comment (ludstuen90) [reply themes/hackshackers-2017/layouts/partials/home-signup.html]: Sounds good - I have removed this header, and also simplified the underlying codebase. Now will look like this when merged in: 
image

Comment (ludstuen90) [reply themes/hackshackers-2017/layouts/partials/home-signup.html]: Sounds good, I have taken out this header, and also simplified the code once more. Now, the change will look like this once merged in: 
image

File: content/_index.md
Comment (burtherman) [inline content/_index.md:1]: I think we don't need to add this sentence - having the actual seal there should be good enough

**Review by burtherman — APPROVED** [review]
All good!

**Pull request merged**